wg-advisors/why/why_discussions_and_decisions.md

```diff
@@ -1,13 +1,77 @@
 # Why discussion and decision making had to happen on the mailing list
 
-TODO:
-
 ## Summary 
 
+Discussions and decision making should happen on the mailing list to ensure that the project is
+transparent and that all decisions are made in the open. This is important to ensure that all
+community members have the opportunity to participate in the decision making process.
+
 ## Links to relevant documents
 
+* [The Apache Way](https://www.apache.org/theapacheway/)
+* [Mailing list guidelines](https://www.apache.org/foundation/mailinglists.html)
+
 ## Why are we doing it?
 
+Mailing list the primary communication channel for the project. It has been chosen - at the dawn of the
+Foundation as the primary communication channel because it was the most inclusive and accessible channel
+for all community members, allowing for asynchronous communication and ensuring that all community members
+have the opportunity to participate. All the important decisions and discussions happens there - including
+the decision to invite new committers and PMC members, the decisions to release software, and the decisions
+to change the project's direction.
```

Comment:
Other good reasons for using mailing lists are that they are accessible to everyone in any country, can be archived, are easily searchable, and are asynchronous, so anyone can take part no matter their time zone/day job/other responsibilities. I can see They also have far more information density than online chat or some other forms of communication.

Reply:
Also filtered or grouped by rules.